src/app.rs

```diff
@@ -24,15 +24,17 @@ use gtk::prelude::ActionMapExtManual;
 use crate::config::{APP_ID, BASE_ID, RESOURCE_PATH};
 use crate::win::CarteroWindow;
 use crate::windows::SettingsDialog;
-
+static MACOS: &str = "macos";
 #[macro_export]
 macro_rules! accelerator {
     ($accel:expr) => {
-        if cfg!(target_os = "macos") {
-            concat!("<Meta>", $accel)
-        } else {
-            concat!("<Primary>", $accel)
+        let concat_for_accel: String = "<Primary>";
+
+        if cfg!(target_os = MACOS) {
+            concat_for_accel = "<Meta>";
         }
```

Comment:
This is not going to compile. In Rust, it is not legal to change the value of a variable after assignment, because variables are immutable.

It is possible to re-declare a variable using the `let` keyword again, but using let inside an `if` would not work either because then the new variable is scoped to the `let`, so it regains the old value at the end of the block.

Given that these are macros (function names that end with `!` are macros), they will always be inlined during compilation, therefore always translated to a single line of code that is either one `concat!` or the other. Therefore, there is no actual performance gain.